### ISSUE TITLE
[chrome/Edge + Jaws2022] When text area contains value, JAWS does not read the text linked to the textarea via  aria-describedby.

### ISSUE THREAD

**Issue by madhaviherle** [FreedomScientific/standards-support#201]
### Summary

1. Created a basic text area
```
<label for="address">Label</label><br>
<Textarea name="addresstext" aria-describedby="help" id="address">test text area</textarea>
<p id="help"> Help text for text area </p>
```

### Expected result
When tabbed to the text area, screen reader should announce the associated help text.

### Actual result
JAWS 17 and 2018 do not read the help text when text area contains value. Fails in IE 11 and Chrome 73
JAWS 2018 does not read the help text even when placeholder is present in the text area in Chrome 73

#### Operating System and version
Windows 7

#### Browser and version:
IE 11 and Chrome 73

**Comment by JAWS-test** [comment]
The problem also occurs with JAWS 2019 and IE 11. It also occurs with the title attribute, i.e., the description is generally not output in textarea with text content.

**Comment by JAWS-test** [comment]
The problem also exists with JAWS 2019 in combination with Chrome and Firefox

**Comment by stevefaulkner** [comment]
created test cases https://s.codepen.io/stevef/debug/RwbZERv, confirmed bug

**Comment by stevefaulkner** [comment]
This issue has already been filed and assigned internally: Bug 109662 - When textareas have initial default text then accessible descriptions not read in some browsers

**Comment by JAWS-test** [comment]
The problem still occurs with JAWS 2020.2008

**Comment by stevefaulkner** [comment]
thanks @marie450 I have contacted @ggordon-vispero for an update

**Comment by stevefaulkner** [comment]
Found out the internal bug had been closed as it was initially reported against IE11, It has now been re-opened with an increased priority.

**Comment by adamsamec** [comment]
The bug still persists with JAWS 2023.2209.11 BETA. And it occurs with Mozilla Firefox 105 as well.

**Comment by stevefaulkner** [comment]
Works in JAWS 2024.2310.70
[test case](https://codepen.io/stevef/pen/RwbZERv)
